lib/index.coffee

```diff
@@ -51,6 +53,29 @@ streamToString = (stream) ->
 		.on 'end', ->
 			resolve(Buffer.concat(chunks).toString())
 
+setupMetrics = (metrics) ->
+	metrics.describe('gc_space_reclaimed', 'Disk space the GC must free')
+	metrics.describe('gc_number_images_to_remove', 'Number of images to be removed at a given time')
```

Comment:
this should be `metrics.describe.counter` (also, I don't see it recorded anywhere in this PR)

Reply:
It would be, I believe - correct me if I'm wrong - observed at the start of each gc run?

Reply:
It is recorded in L43 and that function is called each time the GC is triggered.

This case is similar to the one above, the number of images to remove depends on the system's current state. For example, it might be that a given time, removing only one image will be enough to free the reclaimed space. But maybe another time it will need to remove four images... so in this case the recorded value might go up and down over time...

Reply:
Following the reasoning above for `gc_space_reclaimed`, this should also be a counter incremented with each GC run.